src/hooks/cgroups/pbs_cgroups.PY

```diff
@@ -3735,6 +3766,61 @@ class CgroupUtils(object):
                 raise
         return self.assigned_resources is not None
 
+    @staticmethod
+    def add_jobid_to_cgroup_jobs(jobid):
+        filename = os.path.join(PBS_MOM_HOME, 'mom_priv', 'hooks', 'hook_data',
+                                'cgroup_jobs')
```

Comment:
Can this filename assignment be made into a global/class variable? I see it repeated 5 times in various functions.

Reply:
Done.